docs/renewables/solar/method.rst

```diff
@@ -0,0 +1,35 @@
+From Solar Irradiance to Power Output
+#####################################
+Naive
+^^^^^
+Power output is estimated using a simple normalization procedure. For each solar plant
+location in the grid model the hourly Global Horizontal Irradiance (GHI) is divided by
+the maximum GHI over the period considered. This procedure is referred to as naive since
+other factors can possibly affect the conversion from solar radiation at ground to power
+such as the temperature at the site as well as many system configuration including
+tracking technology.
+
+
+System Adviser Model
+^^^^^^^^^^^^^^^^^^^^
+The `System Adviser Model <https://sam.nrel.gov/>`_ (SAM) developed by NREL is used to
+estimate the power output of a solar plant. Irradiance data along with other
+meteorological parameters must first be retrieved from NSRDB for each plant site. This
+information are then fed to the SAM Simulation Core (SCC) and the power output is
```

Comment:
This information are --> this information is

Reply:
Done